style.css

```diff
@@ -0,0 +1,149 @@
+body {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  background: #fff;
+  overflow: hidden;
```

Comment:
- [ ] Kindly remove the height and overflow properties from the body. It hides the LET'S CONNECT title and the social links.

Reply:
Thanks for the advise